extension/WebJobs.Extensions.RabbitMQ/Services/IRabbitMQService.cs

```diff
@@ -7,8 +7,6 @@ namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ;
 
 public interface IRabbitMQService
 {
-    IRabbitMQModel RabbitMQModel { get; }
```

Comment:
`IRabbitMQModel` was redundant wrapper around `IModel`. I am not sure what it accomplished since `IModel` is itself an interface and fully mockable for tests. Also `IRabbitMQService` was keeping reference to both `IModel` and wrapper of same class `IRabbitMQModel` and on different times, it called one of the two properties, which totally looked illogical.